draft-ietf-quic-transport.md

```diff
@@ -1579,6 +1579,10 @@ the client during connection establishment with a Retry packet (see
 {{validate-retry}}) or in a previous connection using the NEW_TOKEN frame (see
 {{validate-future}}).
 
+The amount that servers are permitted to send is also constrained by the limit
```

Comment:
Suggestion: "In addition to sending limits imposed prior to address validation, servers are also constrained in what they can send by the limits set by the congestion avoidance algorithm.  Clients are only constrained by the congestion avoidance algorithm."

Reply:
s/congestion avoidance algorithm/congestion controller.

Reply:
@ianswett @janaiyengar Thank you! Applied the suggestions.